controller/content.go

```diff
@@ -0,0 +1,134 @@
+package controller
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"peanut/domain"
+	"peanut/repository"
+	"peanut/usecase"
+	"strconv"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+type ContentController struct {
+	Usecase usecase.ContentUsecase
+}
+
+func NewContentController(db *gorm.DB) *ContentController {
+	return &ContentController{
+		Usecase: usecase.NewContentUsecase(repository.NewContentRepo(db)),
+	}
+}
+
+func (c *ContentController) GetContents(ctx *gin.Context) {
+	contents, err := c.Usecase.GetContents()
+	if err != nil {
+		ctx.JSON(http.StatusNoContent, gin.H{
+			"message": "not found any record",
+		})
+		return
+	}
+	ctx.JSON(http.StatusOK, domain.Response{
+		Success: true, Data: contents, Message: "Get data successful",
+	})
+}
+
+func (c *ContentController) CreateContent(ctx *gin.Context) {
+	thumbnailDir := os.Getenv("UPLOAD_THUMBNAIL_PATH")
+	mediaDir := os.Getenv("UPLOAD_MEDIA_PATH")
+	var req domain.CreateContentRequest
+	if !bindForm(ctx, &req) {
+		return
+	}
+
+	mapInt, err := getIntFieldsReq(req)
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, domain.Response{false, nil, err.Error()})
+		return
+	}
+	mapBool, err := getBoolFieldsReq(req)
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, domain.Response{false, nil, err.Error()})
+		return
+	}
+
+	thumbnailName := time.Now().Format("20060102150405") + req.Thumbnail.Filename
```

Comment:
chỗ này a recommend sinh unique id bằng thuật toán uuid, hoặc ulid để tránh trường hợp bị trùng tên file khi có nhiều request đến cùng lúc